contrib/cloud.google.com/go/pubsub.v1/example_test.go

```diff
@@ -0,0 +1,39 @@
+package pubsub
+
+import (
+	"context"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func ExamplePublish() {
+	ctx := context.Background()
+	client, err := pubsub.NewClient(ctx, "project-id")
+	if err != nil {
+		// TODO: Handle error.
```

Comment:
```suggestion
		log.Fatal(err)
```
Should be fine for the example's sake.